docs/sources/setup-grafana/configure-grafana/_index.md

```diff
@@ -1965,6 +1965,18 @@ Enter a comma-separated list of plugin identifiers to hide in the plugin catalog
 
 <hr>
 
+## [plugin.plugin_id]
+
+This section allows to configure plugin-specific settings. `plugin_id` must be replaced with the plugin id present in `plugin.json`.
```

Comment:
Maybe add something along the lines "properties described in this section are available for all plugins but it should be set independently"